### PR TITLE
[fix #192] fix unified sorter

### LINKED ISSUE #192: The order of the two events may be swapped by unified sort
### 1. Describe the bug

unified sort use heap sort which is unstable algorithm.

### 2. Minimal reproduce step (Required)
1. start cdc
2. batch put same key with different values into src tikv
3. read src tikv and dst tikv
4. get different values

### 3. What did you see instead (Required)
1. get different values

### 4. What did you expect to see? (Required)
1. get same values

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?
fix issue #192

Issue Number: close #192

Problem Description: **TBD**
The unified sort use heap sort which is unstable algorithm, which can  swap two equial events.

### What is changed and how does it work?
Add new field for RawKV to help sorter.

### Code changes

- Has exported function/method change
- Has exported variable/fields change
- Has interface methods change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test

### CODE REVIEW

File: cdc/cdc/model/kv.go [hunk 1]
```diff
@@ -87,11 +87,13 @@ type RawKVEntry struct {
 	// Additional debug info
 	RegionID  uint64 `msg:"region_id"`
 	KeySpanID uint64 `msg:"keyspan_id"`
+	// For sort
```
Comment (pingyu) [inline cdc/cdc/model/kv.go:90]:
```suggestion
	// For providing additional sequence number.
	// To keep `RawKVEntries` from the same region in order during unstable sorting of `sorter`.
	// The sequence number is generated by auto-increment in `puller` node of `processor.pipeline`.
```

File: cdc/cdc/model/mounter.go [hunk 2]
```diff
@@ -69,6 +74,8 @@ func ComparePolymorphicEvents(i, j *PolymorphicEvent) bool {
 		} else if j.IsResolved() {
 			return true
 		}
+
+		return i.Sequence() < j.Sequence()
```
Comment (pingyu) [inline cdc/cdc/model/mounter.go:78]: Suggest to add unit test for `ComparePolymorphicEvents`.

**Review by pingyu — COMMENTED** [review]
Rest LGTM.

**Review by pingyu — APPROVED** [review]
LGTM~

**Comment by pingyu** [comment]
@haojinming PTAL~

**Review by haojinming — APPROVED** [review]

**Pull request merged**